### PR TITLE
Microsoft Azure update step by step article and screenshots for the new Microsoft Azure Portal

### PR DESCRIPTION
**Pull request by sudoloic**
This is an update of the Symfony deployment on Microsoft Azure for the new portal. Screenshots and some steps were outdated.

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍 

@sudoloic you did a great work here! Updating all those screenshots should have taken you a lot of time. We truly appreciate it!

I've added a commit to your pull request to make some long lines shorter. We have a soft-limit of 80 chars per line and some of the original lines were too long. Thanks!

File: deployment/azure-website.rst [hunk 1]
```diff
@@ -421,8 +404,6 @@ application, configure it with the following content:
 
 .. code-block:: xml
 
-    <!-- web.config -->
-    <?xml version="1.0" encoding="UTF-8"?>
```
Comment (xabbuh) [inline deployment/azure-website.rst]: @sudoloic Was this change intended?

Comment (sudoloic) [inline deployment/azure-website.rst]: Yep. IIS was crashing with line 425.

**Comment by xabbuh** [comment]
Thank you @sudoloic.
